File: trunk/acm/python/spoj11.py
#!/usr/bin/env python
# @author   FAN Kai (fankai@net.pku.edu.cn), Peking University  
# @date     May 20 05:18:35 PM CST 2009

def Z(num):
    r = 0
    while num > 0:
        num /= 5
        r += num 
    print r

def mainflow():
    n  = input()
    for i in range(n):
        num = input()
        Z(num)

mainflow()

File: trunk/acm/cc/spoj11.cc
/**
 * @file    11.cc
 * @author  FAN Kai (fankai@net.pku.edu.cn), Peking University 
 * @date    May 20 05:37:53 PM CST 2009
 */


#include <iostream>
using namespace std;

int main() {
    int n, num, r;
    cin >>n;
    for (int i = 0; i < n; ++i) {
        cin >>num;
        r = 0;
        while (num > 0) {
            num /= 5;
            r += num;
        }
        cout <<r <<endl;
    }

    return 0;
}


